### PR TITLE
Requirements improvements

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/926)

### CODE REVIEW

File: cli/src/main/kotlin/commands/RequirementsCommand.kt [hunk 1]
```diff
@@ -100,12 +100,21 @@ object RequirementsCommand : CommandWithHelp() {
...
+                val message = if (tool.getVersionRequirement().toString() == anyVersion) {
+                    "${tool.javaClass.simpleName} requires '${tool.command()}' in no specific version."
                 } else {
-                    println("\t${tool.javaClass.simpleName} has ${tool.getVersionRequirement()} on the version of " +
-                            "'${tool.command()}'.")
+                    "${tool.javaClass.simpleName} has ${tool.getVersionRequirement()} on the version of " +
+                            "'${tool.command()}'."
                 }
+
+                // TODO: State which version was found, but this requires refactoring of CommandLineTool.
```
Comment (mnonnenmacher) [inline cli/src/main/kotlin/commands/RequirementsCommand.kt]: We should also state if a tool can be bootstrapped, if not found.

File: README.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ For reusing already known scan results, ORT can currently use one of the followi
 
 * [Artifactory](https://jfrog.com/artifactory/)
 
-## Usage
+## Tools
```
Comment (mnonnenmacher) [inline README.md]: Commit message nit: "reall ymore" -> "really more"

File: README.md [hunk 3]
```diff
@@ -160,6 +127,39 @@ Please see [GettingStarted.md](./docs/GettingStarted.md) for an introduction to
 
 Please see [Configuration.md](./docs/Configuration.md) for details about the ORT configuration.
 
+## Supported package managers
```
Comment (mnonnenmacher) [inline README.md]: Commit message nit: "how" -> "know"

File: cli/src/main/kotlin/commands/RequirementsCommand.kt (hunk 1 above)
Comment (sschuberth) [reply cli/src/main/kotlin/commands/RequirementsCommand.kt]: Done.

File: README.md (hunk 2 above)
Comment (sschuberth) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (sschuberth) [reply README.md]: Done.

File: cli/src/main/kotlin/commands/RequirementsCommand.kt [hunk 4]
```diff
@@ -100,12 +100,22 @@ object RequirementsCommand : CommandWithHelp() {
...
             println("${category}s:")
             tools.forEach { tool ->
-                if (tool.getVersionRequirement().toString() == anyVersion) {
-                    println("\t${tool.javaClass.simpleName} requires '${tool.command()}' in no specific version.")
+                val message = if (tool.getVersionRequirement().toString() == anyVersion) {
+                    "${tool.javaClass.simpleName} requires '${tool.command()}' in no specific version."
                 } else {
-                    println("\t${tool.javaClass.simpleName} has ${tool.getVersionRequirement()} on the version of " +
-                            "'${tool.command()}'.")
+                    "${tool.javaClass.simpleName} has ${tool.getVersionRequirement()} on the version of " +
```
[the remark this answers is not in the document]
Comment (sschuberth) [reply cli/src/main/kotlin/commands/RequirementsCommand.kt]: Yes, we should consistently rework this throughout all classes separately.

**Comment by sschuberth** [comment]
Merging despite the unrated AppVeyor issue.

**Pull request merged**